keps/sig-scheduling/1451-multi-scheduling-profiles/README.md

```diff
@@ -305,12 +293,160 @@ Note that we don't require a feature gate as users already have to opt-in by usi
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled)
+      
+      No
+
+* **Does enabling the feature change any default behavior?**
+
+  No, as long as a profile with the name of `default-scheduler` is kept.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  
+  Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  N/A.
+
+* **Are there any tests for feature enablement/disablement?**
+
+  There are test (unit and integration) exercising single a multiple profiles.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+
+  The scheduler errors and exits during start up. Existing workloads are not
+  affected.
+
+* **What specific metrics should inform a rollback?**
+
+  Metric "schedule_attempts_total" remaining at zero when new pods are added.
+
+* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+
+  N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features,
+  APIs, fields of API types, flags, etc.?**
+
+  See [KEP 785].
+
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+
+  - Scheduling metrics will have a new field for the profile name.
+  - Pods have scheduling Events with different scheduler names.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to
+  determine the health of the service?**
+  
+  - [x] Metrics
+    - Metric name: `schedule_attempts_total`, `e2e_scheduling_duration_seconds`
+    - Components exposing the metric: `kube-scheduler`
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+
+  Existing SLOs for kube-scheduler would apply, but can be filtered per profile.
+
+* **Are there any missing metrics that would be useful to have to improve
+  observability if this feature?**
+  
+  No.
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+
+  No.
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+
+  No.
+
+* **Will enabling / using this feature result in introducing new API types?**
+
+  No.
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  
+  No.
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  
+  No.
+
+* **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs][]?**
+  
+  No.
+
+* **Will enabling / using this feature result in non-negligible increase of
+  resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  
+  RAM overhead: each profile is kept in memory with instantiated plugins.
+  CPU overhead: There is a map lookup for each pod to obtain the profile.
```

Comment:
Aren't both negligible (comparing to resource usage of single scheduler, even in relatively small cluster (like 10 nodes?)

Reply:
I figured it was worth mentioning where the overhead comes from. The next paragraph states that such overhead is minimal.